### PR TITLE
Add Prater config

### PR DESCRIPTION
**Pull request by prestonvanloon**
**What type of PR is this?**

 Feature

**What does this PR do? Why is it needed?**

Prater is launching next week and the network configuration is required.

**Which issues(s) does this PR fix?**

https://github.com/eth2-clients/eth2-testnets/pull/40

**Other notes for review**

:rotating_light: Please review configuration is accurate. If you click approve, it means you cross checked this PR implementation with eth2-testnets PR spec. :rotating_light:

Bootstrap node tracking issue filed: https://github.com/prysmaticlabs/prysm/issues/8612

### CODE REVIEW

File: shared/params/testnet_prater_config.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+// config for Pyrmont.
+func UsePraterConfig() {
+	beaconConfig = PraterConfig()
+}
+
+// PraterConfig defines the config for the
+// Prater testnet.
+func PraterConfig() *BeaconChainConfig {
+	cfg := MainnetConfig().Copy()
+	cfg.MinGenesisTime = 1616508000
```
Comment (terencechain) [inline shared/params/testnet_prater_config.go:28]:
```suggestion
	cfg.MinGenesisTime = 1614588812
```
https://github.com/eth2-clients/eth2-testnets/blob/master/shared/prater/config.yaml#L22

It is `1614588812`, you are using genesis_time + genesis_delay

File: shared/params/testnet_prater_config.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	cfg := BeaconNetworkConfig().Copy()
+	cfg.ContractDeploymentBlock = 4367322
+	cfg.BootstrapNodes = []string{
+		// TODO(8612): Define bootstrap nodes.
+	}
+	OverrideBeaconNetworkConfig(cfg)
+}
+
+// UsePraterConfig sets the main beacon chain
+// config for Pyrmont.
```
Comment (terencechain) [inline shared/params/testnet_prater_config.go:19]:
```suggestion
// config for Prater.
```

File: shared/params/testnet_prater_config.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package params
+
+import (
+	eth1Params "github.com/ethereum/go-ethereum/params"
+)
+
+// UsePraterNetworkConfig uses the Pyrmont specific
```
Comment (terencechain) [inline shared/params/testnet_prater_config.go:7]:
```suggestion
// UsePraterNetworkConfig uses the Prater specific
```

**Review by terencechain — CHANGES_REQUESTED** [review]
Thanks for handling this!

There's incorrect genesis time and few instances of s/pyrmont/prater.

File: shared/params/testnet_prater_config.go (hunk 1 above)
Comment (prestonvanloon) [reply shared/params/testnet_prater_config.go:28]: We don't add these together here. The min genesis time should be min genesis time.

**Comment by prestonvanloon** [comment]
Thanks for the review @terencechain. I left some pasta behind when I copied Pyrmont :rofl:

File: shared/params/testnet_prater_config.go (hunk 1 above)
Comment (prestonvanloon) [reply shared/params/testnet_prater_config.go:28]: Actually, what happened was I copied the value from the PR description. 

Screenshot from 2021-03-16 15-49-07

This genesis time is not the min genesis time and you were correct. Thanks!

**Review by terencechain — APPROVED** [review]

**Review by Oleksandr-prog — APPROVED** [review]

**Pull request merged**